spec/lib/action_kit/helper_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe ActionKit::Helper do
+  before do
+    allow(ENV).to receive(:[]).with("AK_PASSWORD"){ "ak_password" }
```

Comment:
I think this is a positive change. I've created a chore to convert other instances to this format here: https://www.pivotaltracker.com/story/show/105575804